src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.maven.settings.crypto.SettingsDecryptionRequest;
 import org.apache.maven.settings.crypto.SettingsDecryptionResult;
 import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.jetbrains.annotations.NotNull;
```

Comment:
We tend to have these annotations on the package info, and then not the JetBrains variant. Can we remove this here?

Reply:
I removed `@org.jetbrains.annotations.NotNull` and annotated the package as `@org.jspecify.annotations.NullMarked` (as I have seen on another package of the codebase). I hope that is what you had in mind, not being familiar with JSpecify...

Reply:
Stepping in as he should be ~skiing~ snowboarding 🏂  :D Yes, that's what he was asking for ;)